visualization/FairnessDashboard/src/ParityMetrics.ts

```diff
@@ -10,41 +10,29 @@ export enum ParityModes {
     "ratio"
 }
 
-export const ParityOptions: IParityOption[] = [
-    {
-        key: "m1",
-        title: "Method 1",
-        description: "uses science to get the answer",
-        parityModes: [ParityModes.difference, ParityModes.ratio]
-    },
-    {
-        key: "m2",
-        title: "Method 2",
-        description: "guesses at truth",
-        parityModes: [ParityModes.difference, ParityModes.ratio]
-    },
-    {
-        key: "m3",
-        title: "Method 3",
-        description: "guesses at truth",
-        parityModes: [ParityModes.difference, ParityModes.ratio]
+export const ParityOptions: {[key: string]: IParityOption} = {
+    "selection_rate": {
+        key: "selection_rate",
+        title: "Parity difference",
+        description: "Parity differenece",
+        parityModes: [ParityModes.difference]
     },
-    {
-        key: "m4",
-        title: "Method 4",
-        description: "guesses at truth",
-        parityModes: [ParityModes.difference, ParityModes.ratio]
+    "selection_rate_ratio": {
+        key: "selection_rate_ratio",
+        title: "Parity ratio",
+        description: "Parity ratio",
+        parityModes: [ParityModes.ratio]
     },
-    {
-        key: "m5",
-        title: "Method 5",
-        description: "guesses at truth",
+    "zero_one_loss": {
+        key: "zero_one_loss",
+        title: "Error rate difference",
+        description: "Error rate difference",
         parityModes: [ParityModes.difference]
```

Comment:
I'm not certain that having the cache store separate values for ratio vs difference is ideal, since the stored values is the same. Instead, the idea is that these arrays should make clear what modes are available in the UI for a given metric (so selection rate would have parityModes: [ratio, difference] and the others would just have [difference]. The UI control presenting the user options is where you can separate these into different options for the user, but for the data, they are the same.

Reply:
if we're just relying on the UI to decide what operation to do with a given metric, then we may not need this array at all (the initial idea was that wed draw a separate toggle if multiple mpairityModes were available for a selected metric). Having a single dropdown that multiplexes metric and mode in the UI does away with the need to enumerate the available modes, its explicitly enumerated in the dropdown options.